<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleScreen</class>
 <widget class="QWidget" name="TitleScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H.D.D. System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/Icon.ico</normaloff>:/res/Icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>493</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/res/bg.png);
}
QPushButton {
background-color: rgba(0, 0, 0, 255);
border-style: solid;
border-width: 4px;
border-color: rgb(46, 47, 48);
color: white;
}
QPushButton::hover{
	background-color: rgb(165, 194, 0);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>511</width>
      <height>330</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>11</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string>执行空洞深潜任务</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>转到绳网...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>关于</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>退出 H.D.D 系统</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
